src/develop/ui/patterns/web/navigation/wizard.md

```diff
@@ -3,153 +3,108 @@ tags: runtime-traditionalweb;
 
     ![](<images/wizard-image-1.png>)
 
-1. Drag as many WizardItems in the Content placeholder as required. 
+**Create a Wizard with Navigation Buttons**
 
-1. Set the content in the placeholders.
+The following example demonstrates how you can create a three step Wizard with navigation buttons.
 
-1. Set the mandatory values.
+1. Rename each of the Wizard Items to the following: 
+    * Shopping Details
+    * Payment Details
+    * Review Order
 
-    ![](<images/wizard-image-2.png>)
+        ![](images/wizard-image-14.png)
 
-1. Publish and test.
+1. Create an Input Parameter with the following properties:
+    
+    ![](images/wizard-image-13.png)
+ 
+    This input paramenter controls the current Wizard step. 
+    Setting the **Default Value** to 1 ensures the Wizard always starts at step 1.
 
+1. From the Toolbox, drag two Button widgets onto your screen and set their names to **Previous** and **Next**.
 
-## Input Parameters
+1. Define the behaviour for each of the navigation buttons. 
 
-### Wizard
+      ![](images/wizard-image-15.png)
 
-| **Input Name** |  **Description** |  **Type** | **Mandatory** | **Default Value** |
-|---|---|---|---|---|
-| Orientation  |  Set the orientation. | Orientation Identifier | False | Entities.Orientation.Horizontal |
-| ExtendedClass  |  Add custom style classes to this Block. | Text | False | none |
+1. Create an expression that defines the status of each of the Wizard Items (current step, step already completed, step yet to be completed), depending on what stage the user is at.
 
-### Wizard Item
-
-| **Input Name** |  **Description** |  **Type** | **Mandatory** | **Default Value** |
-|---|---|---|---|---|
-| Step  | Set the step. | Step Identifier | True | none |
-| UseTopLabel  |  If true, label is placed above the icon. |  Boolean | False | False |
-| ExtendedClass  |  Add custom style classes to this Block. | Text | False | none |
-  
-## Layout and Classes
+    ![](images/wizard-image-16.png)
 
-![](images/wizard-image-3.png?width=750)
+1. Create a condition for both the **Previous** and **Next** buttons. This ensures that the user can never goes above or below the actual number of steps in the Wizard.
 
-## CSS Selectors
+    ![](images/wizard-image-17.png)
 
-| **Element** |  **CSS Class** |  **Description**  |
-| ---|---|---
-| .wizard |  .wizard.wizard-vertical|  When the wizard orientation is vertical  |
-| .wizard-item |  .wizard-item.active|  Defines the current active step  |
-| .wizard-item |  .wizard-item.past|  Defines the previous step  |
-| .wizard-item |  .wizard-item.next|  Defines the next step  |
+**Add Content to your Wizard**
 
-## Advanced Use Case
+1. Create a condition to control what is displayed, depending on what step is active.
+    
+    ![](images/wizard-image-19.png)
 
-### Use Wizard with ListRecords
+1. Add the relevant content to each for each of the steps.
```

Comment:
But where do we add it? Is there a designated area to add it? Please elaborate.

Reply:
Edited: From the Toolbox, drag the relevant content to each of the Wizard Items in the Wizard